apm-sniffer/config/agent.config

```diff
@@ -90,3 +90,6 @@ logging.level=${SW_LOGGING_LEVEL:INFO}
 
 # Kafka producer configuration
 plugin.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+
+# elclude some plugins which define in plugins dir,
+plugin.exclude_plugins=${SW_EXCLUDE_PLUGINS:""}
```

Comment:
This config item should go to [the doc](https://github.com/apache/skywalking/tree/master/docs/en/setup/service-agent/java-agent#table-of-agent-configuration-properties) as well

Reply:
add doc